lib/rules/no-else-return.js

```diff
@@ -33,7 +35,24 @@ module.exports = {
          * @returns {void}
          */
         function displayReport(node) {
-            context.report({ node, message: "Unnecessary 'else' after 'return'." });
+            context.report({
+                node,
+                message: "Unnecessary 'else' after 'return'.",
+                fix: fixer => {
+                    const sourceCode = context.getSourceCode();
+                    const startToken = sourceCode.getTokenByRangeStart(node.start);
+                    const elseToken = sourceCode.getTokenBefore(startToken);
+                    const source = sourceCode.getText(node);
+                    let fixedSource;
+
+                    if (startToken.type === "Punctuator") {
+                        fixedSource = source.substr(1, source.length - 2);
```

Comment:
By the way, thanks for the quick responses and feedback. I'll give this another go and update. :)